src/to-do.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable quotes */
+const InnerTodo = document.querySelector('.todo-body');
+let isEdit = false;
+let editId = null;
+
+export default class display {
+  static getTodo = () => {
+    let Todo;
+    if (localStorage.getItem('todo') == null) {
+      Todo = [];
+    } else {
+      Todo = JSON.parse(localStorage.getItem('todo'));
+    }
```

Comment:
I suggest you start your class names with an uppercase letter and your variables and function names with lowercase letters. 
 Here you can you can consider using one line of code to get the todo from the local storage
`let todo = JSON.parse(localStorage.getItem('todo')) || []`